packages/website/ts/pages/about/about.tsx

```diff
@@ -240,6 +240,16 @@ const teamRow8: ProfileInfo[] = [
     },
 ];
 
+const teamRow9: ProfileInfo[] = [
```

Comment:
This should not be in the diff right?

Reply:
Correct -- not sure why it's showing. It doesn't show up in a local `git diff` -- should be OK to merge with this showing in the PR diff